WordPress/WordPressTest/NSManagedObject+Fixture.swift

```diff
@@ -0,0 +1,24 @@
+import CoreData
+
+extension NSManagedObject {
+
+    /// Loads the contents of any given JSON file into a new NSManagedObject instance.
+    ///
+    /// This helper method is useful for Unit Testing scenarios.
```

Comment:
How would you feel about removing this note?

After all, we are already in the unit test target 😄